src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,11 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+18.0.0b36
++++++++
+
+* Add option `--acns-performance-acceleration-mode <None|BpfVeth>` to `az aks create/update`
```

Comment:
can it be `--acns-datapath-acceleration-mode`?

Reply:
`--acns-performance-acceleration-mode` matches definition we came up with during API review. Also, this conveys that this is an ACNS performance feature.

Reply:
we kept performance as it encompasses multiple features and datapath is one of it. to me, `acns-datapath-acceleration-mode` sounds better and closer to what it does.